benchmarks/union_find/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+find_package(benchmark 1.5.4 REQUIRED)
+message(STATUS "Found benchmark: ${benchmark_DIR} (version \"${benchmark_VERSION}\")")
```

Comment:
This should probably be unified with `bvh_driver`. I think that maybe we could make benchmark a requirement in the root `CMakeLists.txt` when benchmarks are enabled, because that's what we are doing anyways.